File: an2_sem4/MPP/Homework/lab2/java/domain/src/main/java/ro/ubbcluj/cs/stefnmubb/festivalsellpoint/domain/Bilet.java
package ro.ubbcluj.cs.stefnmubb.festivalsellpoint.domain;
public class Bilet extends Entity<Integer> {
    private String numeCumparator;
    private int nrLocuri;

    private Spectacol spectacol;

    public Spectacol getSpectacol() {
        return spectacol;
    }

    public void setSpectacol(Spectacol spectacol) {
        this.spectacol = spectacol;
    }



    public String getNumeCumparator() {
        return numeCumparator;
    }

    public void setNumeCumparator(String numeCumparator) {
        this.numeCumparator = numeCumparator;
    }

    public int getNrLocuri() {
        return nrLocuri;
    }

    public void setNrLocuri(int nrLocuri) {
        this.nrLocuri = nrLocuri;
    }

    public Bilet(String numeCumparator, int nrLocuri) {
        this.numeCumparator = numeCumparator;
        this.nrLocuri = nrLocuri;
    }
}

File: an2_sem4/MPP/Homework/sapt9/cs/FestivalSellpoint/FestivalSellpoint.Domain/Bilet.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FestivalSellpoint.Domain
{
    public class Bilet : Entity<int>
    {
        public string NumeCumparator { get; set; }
        public int NrLocuri { get; set; }
        public Spectacol Spectacol { get; set; }

        public Bilet(string numeCumparator, int nrLocuri, Spectacol spectacol)
        {
            NumeCumparator = numeCumparator;
            NrLocuri = nrLocuri;
            Spectacol = spectacol;
        }

        public override string ToString()
            => $"Bilet {{ nume={NumeCumparator}, locuri={NrLocuri}, spectacol={Spectacol.Id} }}";
    }
}
